### PR TITLE
Add the meshery-osm chart

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

**Description**

 Add the mesheryosm chart

This PR related to #1782  

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: install/kubernetes/helm/meshery/charts/mesheryosm/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 10009
```
[the remark this answers is not in the document]
Comment (Aisuko) [reply install/kubernetes/helm/meshery/charts/mesheryosm/templates/deployment.yaml]: Done

File: install/kubernetes/helm/meshery/charts/mesheryosm/templates/clusterrole.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "mesheryosm.serviceAccountName" . }}
+  labels:
+{{- include "mesheryosm.labels" . | nindent 4 }}
```
[the remark this answers is not in the document]
Comment (Aisuko) [reply install/kubernetes/helm/meshery/charts/mesheryosm/templates/clusterrole.yaml]: Done

File: install/kubernetes/helm/meshery/charts/mesheryosm/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+name: mesheryosm
+description: Mesheryosm chart for deploying Mesheryosm.
```
[the remark this answers is not in the document]
Comment (Aisuko) [reply install/kubernetes/helm/meshery/charts/mesheryosm/Chart.yaml]: Good point

File: install/kubernetes/helm/meshery/charts/mesheryosm/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+name: mesheryosm
```
[the remark this answers is not in the document]
Comment (Aisuko) [reply install/kubernetes/helm/meshery/charts/mesheryosm/Chart.yaml]: Done

**Comment by Aisuko** [comment]
@leecalcote May you give us some advices? #1800

**Review by leecalcote — APPROVED** [review]
Thank you, @Aisuko 👍

**Pull request merged**